notification/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models, IntegrityError
+from django.utils import timezone
+from player.models import Player
+
+
+class NotificationType(models.TextChoices):
+    WEBSITE = "website", "Website"
+    EVENT = "game-event", "Game-Event"
+    CHAT = "chat", "Chat"
+
+
+class Notification(models.Model):
+    player = models.ForeignKey(Player, on_delete=models.CASCADE)
+    sent_time = models.DateTimeField(default=timezone.now)
+    message = models.TextField(null=True, blank=True)
+    notification_type = models.CharField(
+        max_length=100,
+        choices=NotificationType.choices,
+        default=NotificationType.WEBSITE,
+    )
+    is_read = models.BooleanField(default=False)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=['player_id', 'sent_time'], name='unique notification')
+        ]
+
+    def save(self, *args, **kwargs):
```

Comment:
Could you please explain the purpose behind your changes? From what I observe, you have mainly rewritten the `super.save` method and made slight modifications to the error messages. Typically, when rewriting the save function, I would anticipate additional input validations or other modifications that are not present in the original save method. Can you clarify the intention and any additional functionalities you aimed to incorporate with your changes?

Reply:
I just eanted to assert if we have an 'UniqueConstraint' problem.
I saw now its not necessary at all because we can have two different events that the player is registered for have changed at the same time.
Will Remove this save method.